cachelib/allocator/CacheAllocator-inl.h

```diff
@@ -426,17 +426,16 @@ CacheAllocator<CacheTrait>::allocateInternalTier(TierId tid,
                                                  uint32_t expiryTime,
                                                  bool fromBgThread,
                                                  bool evict) {
-  util::LatencyTracker tracker{stats().allocateLatency_};
-
+  util::LatencyTracker tracker{stats().allocateLatency_, static_cast<size_t>(~fromBgThread)};
```

Comment:
@igchor LatencyTracker() won't track latency requests when batch size is <= 0.
So if the request is from backgroundThread, the latency won't be tracked.

Reply:
But `~fromBgThread` for `fromBgThread == false` is `18446744073709551615`. I assume you meant to use `!`?

Reply:
yes. I rectified it.